fixtures/messy-config/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "messy-config",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "deploy": "wrangler deploy",
+    "dev": "wrangler dev",
+    "start": "wrangler dev",
+    "test": "vitest"
```

Comment:
Thanks for creating this fixture. As it stands I don't think this is running any tests in the CI. We could do with a vitest.config.js and then an actual vitest test that runs Wrangler to trigger the config warnings.

Reply:
I'm not sure a full e2e test is warranted for this change. I realise I may have misunderstood the intent behind the fixtures when creating this. I treated it as more of an examples folder where I could put some code.

If you think it's best I'm happy to add one but looking through the existing tests in the repo a unit test [here](https://github.com/cloudflare/workers-sdk/blob/main/packages/wrangler/src/__tests__/deploy.test.ts) checking that the output contains the warnings would give me reasonable confidence in this change.

Thoughts?